index.js

```diff
@@ -111,6 +111,7 @@ function Peer (opts) {
     }
   }
 
+  self._previousStreams = []
```

Comment:
This should be higher up, next to where the other private variables are declared, IMO.